packages/react-native/ReactAndroid/src/main/java/com/facebook/react/internal/views/text/fragments/TextFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.facebook.react.internal.views.text.fragments
+
+import com.facebook.react.views.text.TextAttributeProps
+
+/**
+ * Interface for a text fragment
+ */
+interface TextFragment {
+  fun getTextAttributeProps(): TextAttributeProps
+
+  fun getString(): String?
```

Comment:
Can those and other become properties of this interface?

Reply:
Sure, I missed it. Done.